src/automation/azext_automation/manual/_params.py

```diff
@@ -162,7 +162,6 @@ def load_arguments(self, _):
         c.argument('operating_system', arg_type=get_enum_type(OperatingSystemType),
                    help='Operating system of target machines.')
         c.argument('included_update_classifications',
-                   arg_type=get_enum_type(WindowsUpdateClasses),
```

Comment:
May I ask why do you want to remove the `arg_type`?

Reply:
The logic is incorrect - currently a comma separated list is accepted and each value should match one of the enum values. Also, there is additional logic on what combination of values are allowed - so removed thsi check altogether so that our service can do the validation.
The current behavior just expects one value matching the enum and errors out when multiple are passed - thus blocking customers to use this correctly.